spec/std/hash_spec.cr

```diff
@@ -191,6 +191,126 @@ describe "Hash" do
     end
   end
 
+  describe "update" do
+    it "updates the value of an existing key with the given block" do
+      h = {"a" => 0, "b" => 1}
+
+      h.update("b") { |v| v + 41 }
+      h["b"].should eq(42)
+    end
+
+    it "updates the value of an existing key with the given block (big hash)" do
+      h = {} of Int32 => Int32
+      100.times do |i|
+        h[i] = i
+      end
+
+      h.update(2) { |v|
+        x = v * 20
+        x + 2
+      }
+      h[2].should eq(42)
+    end
+
+    it "returns the old value when key exists" do
+      h = {"a" => 0}
+
+      h.update("a") { |v| v + 1 }.should eq(0)
+    end
+
+    it "returns the old value when key exists (big hash)" do
```

Comment:
Now that the implementation doesn't depend on whether the hash is big or small, I think we can remove all the "(big hash)" specs

Reply:
Yeah, we were discussing the same thing with @straight-shoota 

I agree it looks like they are not needed but the new implementation is based on `find_entry_with_index` (which depends on hash size) and I couldn't find "(big hash) specs" that (indirectly) tests `find_entry_with_index` or `find_entry`

So maybe we should leave these specs? 🤔

Reply:
I think it's fine